static/js/hotkey.js

```diff
@@ -515,6 +515,11 @@ exports.process_hotkey = function (e, hotkey) {
         }
     }
 
+    // Prevents up and down navigation along the message list when lightbox is open/
+    if ((event_name === 'up_arrow' || event_name === 'down_arrow') && overlays.lightbox_open()) {
+        return true; // True - inorder to prevent the default action of browser as well.
```

Comment:
Got the first commit. Thats seems more appropriate and according to the standard.
But about the second comment. Are you talking about the comment? If the preventDefault works I think I can remove the comment and return the appropriate return value false.